wgpu-types/src/lib.rs

```diff
@@ -611,6 +641,7 @@ impl Default for CullMode {
 pub struct RasterizationStateDescriptor {
     pub front_face: FrontFace,
     pub cull_mode: CullMode,
+    pub polygon_mode: PolygonMode,
```

Comment:
This member should have a comment on it saying it can only be non-fill with the feature on.

Reply:
Good point, I added the docs.